ro-base.obo

```diff
@@ -5408,6 +5413,16 @@ is_a: RO:0040036 ! has anatomical participant
 created_by: pg
 creation_date: 2020-08-27T08:13:59Z
 
+[Typedef]
+id: RO:0012008
+name: results in fusion of
+subset: http://purl.obolibrary.org/obo/valid_for_go_annotation_extension
+subset: http://purl.obolibrary.org/obo/valid_for_go_ontology
+subset: http://purl.obolibrary.org/obo/valid_for_gocam
+is_a: RO:0002592 ! results in organization of
+created_by: pg
```

Comment:
can this field take an orcid ID instead?

Reply:
It could, but there is a bit of inconsistency right now in the world of `created_by`.. Not an issue for this release, but we should standardise on OBO level.